templates/keystoneapi/config/keystone.conf

```diff
@@ -11,6 +11,17 @@ enabled=true
 memcache_servers={{ .memcachedServers }}
 tls_enabled={{ .memcachedTLS }}
 
+{{if .enableFederation }}
+[federation]
+trusted_dashboard={{ .federationTrustedDashboard }}
+
+[openid]
+remote_id_attribute={{ .federationRemoteIDAttribute }}
+
+[auth]
+methods = password,token,oauth1,mapped,application_credential,openid
+{{ end }}
```

Comment:
do we need to add this using an api parameter? we do not want to add tht parameters via the service operators api. a user could use the native interface with customServiceConfig to customize it.